defs.bzl

```diff
@@ -154,6 +154,22 @@ def _parse_artifact_str(artifact_str):
     else:
         return parse.parse_maven_coordinate(artifact_str)
 
+JAVA_EXPORT_REPO_NAME = "internal_java_exports_maven"
```

Comment:
Prefix with `_INTERNAL` to make it clear that this is not meant for consumption by end users.

Reply:
I changed to `INTERNAL_JAVA_EXPORT_REPO_NAME` since vars starts with underscore are internal and can't be imported in the build file.